### ISSUE TITLE
Returning true in "StartChat' hook does not prevent the default chatbox from requesting focus.

### ISSUE THREAD

**Issue by ghost** [Facepunch/garrysmod-issues#855]
``` lua
hook.Add("StartChat", "bug_demo", function()
    timer.Simple(0.1, function()
        print(vgui.GetKeyboardFocus())
    end)
    return true
end)
```

**Output**: Panel: [name:ChatInput][class:TextEntry][43,0,632,18]
**Client version**: 14.01.22

**Comment by Earu** [comment]
Can this be fixed please? It's still happening on all branches as of today.

**Comment by robotboy655** [comment]
Changing this would break chatboxes that rely on this, i.e. they use the 3 hooks to simply repaint the chatbox, or at the very least capture the text being typed.

**Comment by Earu** [comment]
Do you mean for modifying the default chatbox panels?

**Comment by robotboy655** [comment]
No, to capture the text inside the default one that is invisible to use in a HUDPaint chatbox skin or whatever, `StartChat` => `ChatTextChanged` => `FinishChat`
